src/main/java/com/google/devtools/build/lib/bazel/BazelRepositoryModule.java

```diff
@@ -394,9 +394,10 @@ public void beforeCommand(CommandEnvironment env) throws AbruptExitException {
                 Event.warn(String.format("Error parsing the .netrc file: %s.", e.getMessage())));
       }
       try {
+        String configPath = repoOptions.downloaderConfig == null ? null : repoOptions.downloaderConfig.getPathString();
         UrlRewriter rewriter =
             UrlRewriter.getDownloaderUrlRewriter(
-                env.getWorkspace(), repoOptions.downloaderConfig, env.getReporter());
+                env.getWorkspace(), configPath, env.getReporter());
```

Comment:
Nit: Could you change the signature of this method to accept a `PathFragment` instead of a `String`?